codenames-front-end/src/components/game/stopped/menu/CNMaster/CNMaster.module.css

```diff
@@ -0,0 +1,59 @@
+
+
+.TeamMaster{
```

Comment:
use styled components instead of modules to handle css styles

Reply:
Explain why you need to use styled components because I don't really understand why it is better